addon/components/document-details.hbs

```diff
@@ -77,6 +77,49 @@
         </span>
       </span>
 
+      <p class="uk-flex">
+        {{#if this.editDescription}}
+          <textarea
+            class="uk-textarea"
+            {{on "input" this.updateDocumentDescription}}
```

Comment:
I think instead of updating the value on input you might want to only update once you click on the checkmark

Reply:
Yeah. Traditionally, I don't do anything on input/change and fetch the values from the form inputs. With SPAs I often saved the values in some private property.

I already started like that here but setting the correct value wasn't as trivial/clean as I hoped. That's when I tried the verison with updating the @document which turned out to work.